src/binary-reader-ir.cc

```diff
@@ -891,12 +891,22 @@ Result BinaryReaderIR::OnCallRefExpr() {
 }
 
 Result BinaryReaderIR::OnReturnCallExpr(Index func_index) {
+  if (current_func_) {
+    // syntactically, a return_call expr can occur in an init expression
+    // (outside a function)
+    current_func_->features_used.tailcall = true;
```

Comment:
Could you clarify here/the comment on why this is an indication of tail calls being used. I didn't quite follow the current comment

Reply:
ah gotcha